### PR TITLE
Some development

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

**Comment by normanjaeckel** [comment]
@emanuelschuetze @jwinzer I can rebase some commits (all contiguous commits of the same autor to one commit) or just do nothing and then merge this to master. What do you think?

**Comment by emanuelschuetze** [comment]
Yes, please rebase some commits and merge it into upstream master.

**Comment by normanjaeckel** [comment]
@emanuelschuetze I rebase some commits and added an extra commit 7540e69f2297511a3790d1166cfbaf7d42c77ecd. Please check (only) this. Then merge.

**Comment by normanjaeckel** [comment]
Maybe tell me some notes I can add to the CHANGELOG.

File: README.rst [hunk 1]
```diff
@@ -115,8 +115,10 @@ Changelog
 
 Version 2.0.0 (unreleased)
 --------------------------
-* Added support for OpenSlides 2.x.
+* Added support for OpenSlides 2.1.x.
 * Added support for speaker list.
+* Added support for assignment polls.
+* Added possibility to change the seat number.
```
Comment (emanuelschuetze) [inline README.rst]: Suggestion for the 2.0.0 section:
* Added support for OpenSlides 2.1.x with autoupdate for all incoming votes.
* New speakers voting (add/remove from list of speakers).
* New election voting (Yes/No/Abstain for one candidate or 1 of n candidates via keypad keys 1..10) with anonymize votes
* Updated motion voting, with anonymize votes.
* Added possibility to change the seat label.

**Comment by emanuelschuetze** [comment]
Some remarks:
* Please change in README: `'Configuration > VoteCollector'.` to `'Settings > VoteCollector'`.
* suggestion: use requirements.txt only and remove requirements_production.txt
* remove DEVELOPMENT.rst (I can't see why we should keep this file. In future we should work with issues)
* @jwinzer Does the generic download link always pointed to the last stable version of VoteCollector (now 1.7.6)? http://files.softwein.de/VoteCollector.zip

**Comment by jwinzer** [comment]
Let’s use a generic link specific to OpenSlides such as
http://files.softwein.de/OpenSlides-VoteCollector.zip

If you want pick a different name.

File: README.rst (hunk 1 above)
Comment (normanjaeckel) [inline README.rst]: Changed.

**Comment by normanjaeckel** [comment]
All done.

**Pull request merged**